app/graphql/mutations/users/update_user.rb

```diff
@@ -4,15 +4,29 @@ class UpdateUser < ::Mutations::BaseMutation
       argument :id, Integer, required: true
       argument :email, String, required: false
       argument :username, String, required: false
+      argument :target_id, String, required: false
+      argument :native_id, String, required: false
 
       type Types::UserType
 
       def resolve(arguments)
         user = User.find(arguments[:id])
         arguments.delete(:id)
+        target_id = arguments.delete(:target_id)
+        native_id = arguments.delete(:native_id)
         user.update(arguments)
+        create_target(target_id, user) if target_id
+        create_native(native_id, user) if native_id
         user
       end
+
+      def create_target(target, user)
+        user.user_languages.create(fluency_level: 1, language_id: target)
+      end
+
+      def create_native(native, user)
+        user.user_languages.create(fluency_level: 0, language_id: native)
+      end
```

Comment:
Just to clarify we wanted to be able to create userLanguages? Or did we want to have userLanguageId passed to update a specific target or native userLanguage?

Reply:
Will a user ever have more than 1 target or native language? I could update this code to say 'grab the user's target language and change it'.

Reply:
Updated to reflect previous comment